src/sentry/search/events/datasets/metrics.py

```diff
@@ -276,3 +287,44 @@ def _resolve_percentile(
             ],
             alias,
         )
+
+    def _resolve_web_vital_function(
+        self,
+        args: Mapping[str, Union[str, Column, SelectType, int, float]],
+        alias: str,
+    ) -> SelectType:
+        column = args["column"]
+        metric_id = args["metric_id"]
+        quality = args["quality"].lower()
+
+        if column not in [
+            "measurements.lcp",
+            "measurements.fcp",
+            "measurements.fp",
+            "measurements.fid",
+            "measurements.cls",
+        ]:
+            raise InvalidSearchQuery("count_web_vitals only supports measurements")
+
+        measurement_rating = Column(f"tags[{indexer.resolve('measurement_rating')}]")
```

Comment:
Should this be some kind of helper function?

Reply:
:+1: yeah dunno why i forgot about using the builders column resolver here